.github/workflows/gofmt.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
     - uses: actions/setup-go@v5
       with:
-        go-version: '1.23'
```

Comment:
Might want to pin to 1.24.1 to avoid some bugs: https://github.com/golang/go/issues?q=sort%3Aupdated-desc+milestone%3AGo1.24.1+label%3ACherryPickApproved